src/sorting.h

```diff
@@ -0,0 +1,16 @@
+/**
+ * @file sorting.h
+ * prototypes for sorting.c
+ *
+ * @author  Ben Gardner
+ * @license GPL v2+
+ */
+#ifndef SORTING_H_INCLUDED
+#define SORTING_H_INCLUDED
+
+#include "uncrustify_types.h"
+
+
+void sort_imports(void);
```

Comment:
Here some functions are missing ... or did we decide not to include some? 
Why is 'uncrustify_types.h' included here?